polaris-core/src/main/resources/schemas/policies/system/data-compaction/2025-02-03.json

```diff
@@ -0,0 +1,41 @@
+{
+  "license": "Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)",
```

Comment:
Json doesn't support comments. Using a field to indicate Apache License instead of a header.